drivers/softlayer/driver_test.go

```
@@ -88,3 +90,9 @@ func TestSetConfigFromFlagsSetsImage(t *testing.T) {
 		t.Fatalf("expected 'MY_TEST_IMAGE'; received %s", img)
 	}
 }
+
+func TestHostnameDefaultsToMachineName(t *testing.T) {
+	d, _ := getTestDriver()
```

Why eat the error here?